CHANGELOG.md

```diff
@@ -271,7 +275,9 @@ which enable `dep` dependencies to be cached in jobs as long as `Gopkg.lock` doe
 
 
 
-[Unreleased]: https://github.com/giantswarm/architect-orb/compare/v0.8.7...HEAD
+[Unreleased]: https://github.com/giantswarm/architect-orb/compare/v0.8.8...HEAD
+
```

Comment:
```suggestion
```

Reply:
I do these on purpose to have the visual separation of the different semantics and functions of the links. The very first is not to compare. The last is against HEAD. Makes it easier to work with for the humans I thought. Let me know when this is not desirable.